src/common/logging.ts

```diff
@@ -0,0 +1,26 @@
+declare const DEVELOPMENT: boolean;
+
+let hasWarnedDev = false;
+
+/**
+ * Log an error in development mode.
+ * Will warn if the .env file doesn't have DEVELOPEMENT equal to 'true'.
+ *
+ * Should be used to warn that something went wrong.
+ *
+ * @param e the error object to include. A message can be included in the object.
+ */
+function logError(e: Error): void {
+  if (DEVELOPMENT) {
+    console.error(e);
+  } else {
+    if (!hasWarnedDev) {
+      // Message warns the dev that logging is disabled
+      console.warn("Developer Logging Not Enabled");
+      hasWarnedDev = true;
+    }
```

Comment:
So production will still output a warning once if a `logError` line is used?

Reply:
Yeah! The original intent was I wanted to log a single time to let the developer know that logging was disabled.
I wanted there to be some warning that logging is disabled, to indicate that console silence doesn't necessarily mean that the app is running smoothly. I figured that since this is only showing once, it acts as a fair balance between user friendliness and developer convenience

I'm going to assume that there might holes in my initial plan, so I have now changed this. You will now only get messages in development mode